### PR TITLE
Replaces "Bread" smite with "Objectify"

### PR DESCRIPTION
**Pull request by Jacquerel**
## About The Pull Request

dreamseeker_BHBiusguBf

Instead of turning people into bread, the bread smite can now turn people into any object (including bread).
Transforming people into most objects will entomb them inside that object as normal.
"Transforming" someone into a crate or closet will trap them inside until someone opens the closet, at which point they are freed.
"Transforming" someone into a mob will place them inside the mob as an unwitting observer who can only watch as it does whatever that mob does.

If you transform someone into something with a larger than 32x32 icon it will look weird, but its an admin tool so do that at your own risk.

## Why It's Good For The Game

The worker must have bread, but she must have roses, too,

## Changelog

:cl:
admin: Admins can transform misbehaving players into arbitrary objects at will.
/:cl:

### CODE REVIEW

**Comment by nikothedude** [comment]
morty

morty im

**Comment by CRITAWAKETS** [comment]
finally i can objectify- [my lawyers have advised me not to continue this joke]

**Review by Ghommie — APPROVED** [review]

File: code/datums/components/itembound.dm [hunk 1]
```diff
@@ -11,19 +11,39 @@
...
+/datum/component/itembound/UnregisterFromParent()
+	REMOVE_TRAIT(parent, TRAIT_INCAPACITATED, SMITE_TRAIT)
+	if (isliving(parent))
+		var/mob/living/living_parent = parent
+		living_parent.remove_status_effect(/datum/status_effect/grouped/stasis, STASIS_ADMIN)
+	return ..()
+
+/datum/component/itembound/proc/on_examined(atom/source, mob/user, list/examine_list)
+	SIGNAL_HANDLER
+	examine_list += span_notice("If you hold it up to your ear, you can hear the screams of the damned.")
```
Comment (AnturK) [inline code/datums/components/itembound.dm:34]: I'd put it on the close examine personally.

File: code/modules/admin/smites/become_object.dm [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+	name = "Become Object"
+	/// What are we going to turn them into?
+	var/atom/transform_path = /obj/item/food/bread/plain
+
+/datum/smite/objectify/configure(client/user)
+	var/attempted_target_path = input(
+		user,
+		"Enter typepath of an atom you'd like to turn your victim into.",
+		"Typepath",
+		"/obj/item/food/bread/plain",
```
Comment (AnturK) [inline code/modules/admin/smites/become_object.dm:14]: use type to text, for the inevitable another food repathing

**Pull request merged**